Formula/ignition-tools2.rb

```diff
@@ -0,0 +1,35 @@
+class IgnitionTools2 < Formula
+  desc "Entry point for ignition command-line tools"
+  homepage "https://ignitionrobotics.org"
+  url "https://github.com/ignitionrobotics/ign-tools.git", branch: "main"
+  version "1.999.999~0~20220414"
+  license "Apache-2.0"
+  head "https://github.com/ignitionrobotics/ign-tools.git", branch: "ign-tools2"
```

Comment:
I think we can remove this line, I don't see it on other `main` branch formulae:

https://github.com/osrf/homebrew-simulation/blob/master/Formula/ignition-common5.rb

And the `ign-tools2` branch doesn't exist yet